src/views/Nft/market/Profile/ConnectedProfile.tsx

```diff
@@ -31,7 +25,7 @@ const TabMenuWrapper = styled(Box)`
   }
 `
 
-const ConnectedProfile = () => {
+const ConnectedProfile: FC = ({ children }) => {
```

Comment:
As far as I understood, We will fetch achievements twice when connected user profile viewed right?

Reply:
I will move it to `getLayout`, should preserve state and layout between the navigation.

Reply:
fix fetch twice https://github.com/pancakeswap/pancake-frontend/pull/2978/commits/05605c0da344cb6ebdfdab8514bb979e435a738a